docker-compose.yml

```diff
@@ -72,7 +72,7 @@ services:
       - mockserver
 
   dev-configuration-variables-resolver:
-    <<: *dev82
```

Comment:
Tohle se mi moc nepozdava. Ta lib stale podporuje PHP 8.2 a vyijet by se to urcite melo na nejmensi podporovane verzi, aby si tam nezanesl nejakou nepodporovanou feature.

Reply:
no já jsem to trošku čekal, tak asi začnu od updatu job runneru no :D